templates/download/raspberry-pi/index.html

```diff
@@ -340,100 +312,69 @@ <h2>Supported devices</h2>
+        </li>
+        <li class="p-list__item">
+          <a href="https://maas.io/tutorials/build-your-own-bare-metal-cloud-using-a-raspberry-pi-cluster-with-maas#1-overview">Build your own bare metal cloud using a Raspberry Pi cluster with MAAS</a>
+        </li>
+      </ul>
     </div>
-    <div class="col-3 p-divider__block">
-      <div class="p-heading-icon--muted">
-        <div class="p-heading-icon__header">
-          {{ image (
-            url="https://assets.ubuntu.com/v1/e2315503-tutorial.svg",
-            alt="",
-            width="231",
-            height="150",
-            hi_def=True,
-            loading="lazy",
-            attrs={"class": "p-heading-icon__img p-heading-icon__img--small"}
-            ) | safe
-          }}
-          <h4 class="p-heading-icon__title">Tutorial</h4>
-        </div>
-        <h3 class="p-heading--4"><a href="/blog/homelab-clusters-lxd-micro-cloud-on-raspberry-pi">Homelab clusters: LXD micro cloud on Raspberry Pi</a></h3>
+  </div>
+</section>
+
+<section class="p-section--deep">
+  <hr class="p-rule is-fixed-width"/>
+  <div class="row">
+    <div class="col-3 col-medium-6">
+      <div class="p-section--shallow">
+        <h2 class="p-text--small-caps"><a href="/blog/tag/raspberry-pi">Latest from&nbsp;our&nbsp;blog&nbsp;&rsaquo;</a></h2>
       </div>
     </div>
-    <div class="col-3 p-divider__block">
-      <div class="p-heading-icon--muted">
-        <div class="p-heading-icon__header">
-          {{ image (
-            url="https://assets.ubuntu.com/v1/e2315503-tutorial.svg",
-            alt="",
-            width="231",
-            height="150",
-            hi_def=True,
-            loading="lazy",
-            attrs={"class": "p-heading-icon__img p-heading-icon__img--small"}
-            ) | safe
-          }}
-          <h4 class="p-heading-icon__title">Tutorial</h4>
-        </div>
-        <h3 class="p-heading--4"><a href="https://maas.io/tutorials/build-your-own-bare-metal-cloud-using-a-raspberry-pi-cluster-with-maas#1-overview">Build your own bare metal cloud using a Raspberry Pi cluster with MAAS</a></h3>
+    <div class="col-9 col-medium-6">
+      <div class="row p-section--shallow" id="blog-latest">
       </div>
     </div>
   </div>
+
+  <template style="display:none" id="blog-template">
+    <div class="col-3 col-medium-2">
+      <div class="article-image u-blog-thumbnails p-image-wrapper"></div>
+      <h3 class="p-heading--5"><a class="article-link article-title"></a></h3>
+      <p class="article-excerpt"></p>
+    </div>
+  </template>
 </section>
 
-{% with section_classes='p-strip is-bordered', heading_topic='Raspberry Pi', tag_name='raspberry-pi', tag_id='1672', limit='4', custom_heading="Check out the latest blog" %}
-  {% include "shared/_latest_news_strip.html" %}
-{% endwith %}
+<script src="{{ versioned_static('js/dist/latest-news.js') }}"></script>
```

Comment:
This is legacy code. It is available, otherwise the blog wouldn't work, you would see it broken.

Reply:
I know, but if it's not imported in `build.js` it will not be bundled and won't exist in dist folder. We already had several similar incidents in the past because of that.